src/pear/config.vsh.yaml

```diff
@@ -0,0 +1,145 @@
+functionality:
+  name: pear
+  description: "PEAR is an ultrafast, memory-efficient and highly accurate pair-end read merger"
+  info:
+    keywords: [ "pair-end", "read", "merge" ]
+    homepage: https://cme.h-its.org/exelixis/web/software/pear
+    documentatoin: https://cme.h-its.org/exelixis/web/software/pear/doc.html
+    reference: doi:10.1093/bioinformatics/btt593
+    licence: "['Creative Commons Attribution-NonCommercial-ShareAlike 3.0 Unported']"
+  requirements:
+    cpus: 1
+    memory: 4GB
+    commands: [ pear , gzip ]
+  argument_groups:
+    - name: Inputs
+      arguments:
+        - name: --forward_fastq
+          alternatives: -f
+          type: file
+          description: Forward paired-end FASTQ file
+          required: true
+          example: "forward.fastq"
+        - name: --reverse_fastq
+          alternatives: -r
+          type: file
+          description: Reverse paired-end FASTQ file
+          required: true
+          example: "reverse.fastq"
+    - name: Outputs
+      arguments:
+        - name: --output
+          alternatives: -o
+          type: string
+          description: The prefix of the output file(s). The output files will be named as <output>.assembled.fastq and <output>.unassembled.forward.fastq, <output>.unassembled.reverse.fastq and <output>.discarded.fastq.
+          required: true
+          example: "output.fastq"
```

Comment:
- `direction: output` is missing
- Prefixes won't work. Use one of the following:
    1. Separate files: `--output_assembled`, `--output_unassembled_forward`, `--output_unassembled_reverse` and `--output_discarded`
    2. Simpler files but simpler: `--assembled`, `--unassembled_forward`, `--unassembled_reverse` and `--discarded`. 
    3. Just return a directory `--output foo/` with foo containing `assembled.fastq`, etc.
    
  I propose option 2 :)